waku/protocol/v2/waku_historic_messages.nim

```diff
@@ -0,0 +1,73 @@
+import 
+  libp2p/protocols/protocol,
+  libp2p/protobuf/minprotobuf,
+  stew/results
+
+const
+  HistoricMessagesCodec = "/waku/history/1.0.0"
```

Comment:
For a spike OK, but we need to make sure these work fine together. See https://github.com/vacp2p/specs/issues/87

Probably worth adding -alpha or so here as well and make sure this is in spec